fe/fe-core/src/main/java/com/starrocks/common/StarRocksFEMetaVersion.java

```diff
@@ -12,6 +12,9 @@
     //support hive external read
     public static final int VERSION_3 = 3;
 
+    //support iceberg external read
+    public static final int VERSION_4 = 4;
```

Comment:
Increase the version only when need to compatible with the old metadata.
Iceberg table is a new metadata, so it is not needed.